config/initializers/kaminari_config.rb

```diff
@@ -2,7 +2,7 @@
 
 Kaminari.configure do |config|
   config.default_per_page = 100
-  config.max_per_page = 100
+  config.max_per_page = 100_000
```

Comment:
Is 100,000 a bit big?

Reply:
I was mixed up, thought this was being used in the way `default_per_page` is used. All good!